ci-operator/step-registry/ipi/conf/nutanix/zones/ipi-conf-nutanix-zones-commands.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH="${SHARED_DIR}/install-config-failureDomains.yaml"
+
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/nutanix_context.sh"
+NUTANIX_AUTH_PATH=${CLUSTER_PROFILE_DIR}/secrets.sh
+
+declare prism_element1_host
+declare prism_element1_port
+declare prism_element1_uuid
+declare prism_element1_subnet
+declare prism_element2_host
+declare prism_element2_port
+declare prism_element2_uuid
+declare prism_element2_subnet
+# shellcheck source=/dev/null
+source "${NUTANIX_AUTH_PATH}"
+
+cat >"${PATCH}" <<EOF
+platform:
+  nutanix:
+    defaultMachinePlatform:
+      failureDomains:
```

Comment:
Just be curious, is there some check point for the zone install?

Reply:
Not yet, I'll add check points when adding e2e ci

Reply:
got it.